com.playeveryware.eos/Runtime/Core/Config/Attributes/ConfigFieldAttribute.cs

```diff
@@ -49,6 +49,13 @@ public class ConfigFieldAttribute : Attribute
         /// </summary>
         public ConfigFieldType FieldType { get; }
 
+        /// <summary>
+        /// If the FieldType is ConfigFieldType.Enum, this value is used to 
+        /// determine the type of enum, so that its possible values can be
+        /// enumerated.
+        /// </summary>
+        public Enum EnumType { get; }
```

Comment:
This should not be within the `ConfigFieldAttribute` class, because it is only used when the config field attribute is representing an enum value. Instead, a class should be created for `ConfigFieldAttributes` of type Enum:

```c#
public class EnumFieldAttribute : ConfigFieldAttribute
{
    public Enum EnumType { get; }
    /* ... */
}
```

However, since this PR does not introduce any area where this functionality is utilized, it should be excluded from this PR.

Reply:
I can agree to that focus. I think there's *some merit* in including it because it makes clear that this is something that needs support when we do implement that, but it's in your brain now. Will exclude. Good point on it not being applicable to other kinds of field attributes.